pkg/helpers.go

```diff
@@ -0,0 +1,127 @@
+package sfu
+
+import (
+	"encoding/binary"
+)
+
+// VP8Helper is a helper to get temporal data from VP8 packet header
```

Comment:
Should we contribute this to https://github.com/pion/rtp/blob/master/codecs/vp8_packet.go ?

Reply:
This just take care of the temporal data, it doesn't care of other flags or make a copy a of the vp8 payload for performance. So it's out of scope of the rtp/codecs. I can make a pull request to add tempral flags on vp8 codec, but whe should not use it on sfu for temporal info.